### ISSUE TITLE
[Bug]: Update homebridge version required in package.json

### ISSUE THREAD

**Issue by Danimal4326** [mrjackyliang/homebridge-adt-pulse#145]
### Pre-check confirmation

- [X] I have provided a descriptive title for the bug I am currently having
- [X] I have kept the original title prefix, __"[Bug]: "__
- [X] I have read the `README.md` file, and this is __NOT__ a duplicate bug report
- [X] I am __NOT__ requesting a new feature
- [X] I am not requesting help that is related to a support request
- [X] I am using the latest version of the plugin

### Your email address

N/A

### Tell me exactly what happened

Warning showing in logs:

```
The plugin "homebridge-adt-pulse" requires a Homebridge version of ~1.7.0 which does not satisfy the current Homebridge version of 1.8.2. You may need to update this plugin (or Homebridge) to a newer version. You may face unexpected issues or stability problems running this plugin.
```

### Plugin version

3.3.1

### Homebridge version

1.8.2

### Node.js® version

20.14.0

### Output of the "test-api" command

```Shell
N/A
```

### Captured logs relevant to this report

```Shell
The plugin "homebridge-adt-pulse" requires a Homebridge version of ~1.7.0 which does not satisfy the current Homebridge version of 1.8.2. You may need to update this plugin (or Homebridge) to a newer version. You may face unexpected issues or stability problems running this plugin.
```
```


### Post-check confirmation

- [X] I have filled in the form __exactly as requested__ and did not attempt to circumvent form validation

### Legal Agreements

- [X] I agree to the Terms of Use and Privacy Policy

**Comment by mrjackyliang** [comment]
Are you able to ignore the warning and continue the use of the plugin for now? I am progressing on the next update and would like to just push once.

**Comment by Danimal4326** [comment]
I can, I’ll wait for the next release.   Just wanted to file a report in case you missed it.

**Comment by mrjackyliang** [comment]
Thanks for the heads up! Will add that to my tasks

**Comment by mrjackyliang** [comment]
This should now be fixed in `v3.3.5`!
